docs/Text.md

```diff
@@ -32,8 +32,9 @@ fixed_text = get_display(reshape(some_text))
 ```
 
 ### Character or Word Based Line Wrapping
-By default, `multi_line()` and `write()` will wrap lines based on words, using space characters and soft hyphens as seperators.
-For languages like Chinese and Japanese, that don't usually seperate their words, character based wrapping is more appropriate.
+By default, `multi_line()` and `write()` will wrap lines based on words, using space characters and soft hyphens as separators.
```

Comment:
On that occasion, you could also fix an old mistake, and change the incorrect `multi_line()` in that sentence to `multi_cell()`. :wink: